vm-setup/roles/v1aX_integration_test/vars/main.yml

```diff
@@ -65,6 +65,32 @@ IMAGE_NAME_CENTOS: "{{ lookup('env', 'IMAGE_NAME') | default('CENTOS_8.2_NODE_IM
 RAW_IMAGE_NAME_CENTOS: "{{ lookup('env', 'IMAGE_RAW_NAME') | default('CENTOS_8.2_NODE_IMAGE_K8S_v1.18.8-raw.img', true)}}"
 IMAGE_LOCATION_CENTOS: 'https://artifactory.nordix.org/artifactory/airship/images/k8s_v1.18.8/'
 
+# Ironic TLS environment variables
+IRONIC_CA_CERT_B64: "{{ lookup('env', 'IRONIC_CA_CERT_B64')}}"
```

Comment:
Actually we don't need them here (except BMOPATH, GOPATH) I assume, because you are exporting them through ironic_tls_setup.sh and ironic_basic_auth.sh. And for the arguments in `Install Ironic` task, I think it is possible to add lookup for each variable so that you can get the values of those vars from the shell after they are exported via ironic_tls_setup.sh and ironic_basic_auth.sh.

Reply:
Arguments for `install ironic` are not present in `ironic_tls_setup.sh` nor `ironic_basic_auth.sh` IMO.

Reply:
Yes sorry I missed that. But how about other vars?

Reply:
I think we don't need the others, your are right